.gitlab-ci.yml

```diff
@@ -605,7 +597,7 @@ weldr-distro-dot-notation+aliases:
   parallel:
     matrix:
       - RUNNER:
-          - aws/rhel-8.8-ga-x86_64
```

Comment:
I think this one needs to stay on 8.8 because it's testing specific things with aliases.

Reply:
ah thanks